README.md

```diff
@@ -42,6 +42,24 @@ $ cd chanjo-report
 $ pip install --editable .
 ```
 
+### Docker
```

Comment:
Dang, we did the same thing here!

Reply:
😫 And even chatted about it beforehand.. I'm sure I said I would fix docs, add to compose, and answered affirmatively to fixing the ubuntu workflow version when I'm at it. You said you were going to look at the other env, so I suppose for Scout, chanjo etc? 😊

Reply:
Ah no, I meant that I was doing this 😆